src/protocols/secure_channel/SessionEstablishmentExchangeDispatch.h

```diff
@@ -39,10 +36,9 @@ class SessionEstablishmentExchangeDispatch : public Messaging::ExchangeMessageDi
 
     virtual ~SessionEstablishmentExchangeDispatch() {}
 
-    CHIP_ERROR Init(Transport::BLE * transport, TransportMgrBase * transportMgr)
+    CHIP_ERROR Init(TransportMgrBase * transportMgr)
     {
-        ReturnErrorCodeIf(transport == nullptr && transportMgr == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
-        mBLETransport = transport;
+        ReturnErrorCodeIf(transportMgr == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Since we are moving BLE down the stack, the new implementation will enable CRMP for BLE based rendezvous as well. Maybe it's ok to leave it enabled for now. However, we should file an issue to track it.

Reply:
Seems we should also check for TCP transport -> Only enable CRMP on UDP connections for session establishment.